charters/DocumentationCharter.md

```diff
@@ -0,0 +1,54 @@
+
+## Mission and Goals
+
+The Docs Working Group serves as a support, helper, and advisory body to Jupyter subprojects on all aspects of documentation. The core pillars of our mission:
+
+- Improve all aspects of documentation across the Jupyter ecosystem
+- Make high quality documentation that is clear, comprehensive, inclusive, and serves the varying needs of Jupyter's diverse community
+- Engage with the community to help users with Jupyter products, discover their needs, and connect them with relevant information, expertise and resources
+
+The Docs Working Group will provide a place for consistent, focused, holistic efforts to be spent on docs across the whole ecosystem. This group exists in part to provide capacity and resources to the subprojects (some of which are already suffering from a lack of resources/capacity, and more specifically to work on docs in particular).
+
+We want all users to have positive experiences inside the Jupyter ecosystem, especially users who are learning and coming in for the first time, and users with disabilities.
+
+## The Docs Working Group's Activities
+
+The Docs Working Group will focus on the efforts described below, in service of its mission:
+
+- Help write docs (inside the bounds of each subproject's governance)
+    - This includes meta docs, cross-cutting docs (items relevant to multiple subprojects), developer and contributor docs, non user-facing docs and others
+- Develop recommendations and guidance:
+	- For communicating common information and concepts across the Jupyter ecosystem
+        - Because we want to encourage consistency across the Jupyter ecosystem and between different projects
+	- For style and best practices when authoring documentation media
+        - Including technical advice, such as tools-usage and markdown guidance that helps support high level goals, like interlinking/cross-connection between subprojects
+- Community engagement
+	- To help connect users with information, expertise and resources
+	- To gather feedback from the community about what needs to be documented/what information they need
+- Help support and improve all aspects of documentation across the Jupyter ecosystem
+	- This means, for members of the group, also taking on any related (non-document) tasks that will be beneficial to docs and support the Docs Working Group's goals, such as:
+        - Developing application extensions related to docs, like in-app integrated docs, guided in-application tours or hoverable "what's this" features
+        - Automations for gathering ReadTheDocs traffic stats to gauge user interest
+        - Creating pip installable packages for getting offline docs
+        - Writing PRs for unit tests related to documentation code
+        - Automated discourse or CI bots for gathering user feedback from community sites
+        - Adding a new embedded "Hot Topics" feature on ReadTheDocs FAQ pages to automated discourse community discussion aggregator
```

Comment:
I don't have an objection to this at all, but I'd remove it from the charter and instead create an issue on the docs working group repo and handle it as a task instead of a chartered mandate.